Lib/Src/CThermistor.cpp

```diff
@@ -0,0 +1,54 @@
+/**
+ * @file CThermistor.cpp
+ */
+
+/*
+ *  Created on: Feb 15, 2022
+ *      Author: niuslar
+ */
+
+#include "CThermistor.h"
+#include "math.h"
+
+#define OUT_OF_RANGE_VALUE (999.9)
```

Comment:
You should make this 
```
public: 
static const CThermistor::OUT_OF_RANGE = 999.9;
```
That way it can be referenced outside the class.
```
if(thermistor.getTemperature(voltage) == CThermistor::OUT_OF_RANGE)
{
    log.send("sensor fault");
}
```